app/configuration/Config.scala

```diff
@@ -22,4 +23,5 @@ object Config {
     (idWebAppUrl / "signin") ? ("returnUrl" -> s"$contributeUrl$uri") & idSkipConfirmation
   private val idSkipConfirmation: (String, String) = "skipConfirmation" -> "true"
 
+  val guardianDomain = ".theguardian.com"
```

Comment:
I see no problem with an extra config attribute here, especially as we're dealing with two separate concepts: the main website and the contribution page